README.md

```diff
@@ -60,17 +60,50 @@ Now with Github deploys!
 1. The callback URL should look something like `http://192.168.99.100:5000/complete/github`
 1. Make sure there is no trailing slash on the callback URL
 1. Set the `Client ID` and `Client Secret` in your config above
-1. Confirm configuration is working by visiting your running app in a browser (e.g. `http://192.168.99.100:5000/`) and logging in with github 
-### Obtain a Github OAuth Token -> Soon to be depricated. Social Signin will do this for us
+
+### Configure an oAuth application for franklin
+1. create a superuser `docker-compose run web python manage.py createsuperuser`
+1. Login as super user as `http://192.168.99.100:5000/admin/`
+1. Navigate to `Oauth2_Provider --> Applications --> Add Application +`
+1. `user` -> superuser
+1. `Client Type` -> `Confidential`
+1. `Authorization Grant Type` -> `Resource Owner password-based`
+1. Save the `client id` and `client secret`
+
+### Obtain a Github OAuth Token
```

Comment:
We'll need to do another round of you guys testing these docs to make sure everything is easy to setup. With security now enabled, nothing is going to work for you without properly setting up Github app creds, person github token, and franklin oAuth.

If it becomes too much of a bear to develop with, I'll write some sort of script to do all of this for you on local.